[2023-09-30T07:47:53.779+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=dags_python_with_task_group/run_id=scheduled__2023-09-29T07:47:40.794201+00:00/task_id=first_group.inner_function1 permission to 509
[2023-09-30T07:47:53.784+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=dags_python_with_task_group/run_id=scheduled__2023-09-29T07:47:40.794201+00:00/task_id=first_group.inner_function1 permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dags_python_with_task_group/run_id=scheduled__2023-09-29T07:47:40.794201+00:00/task_id=first_group.inner_function1'
[2023-09-30T07:47:53.971+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=dags_python_with_task_group/run_id=scheduled__2023-09-29T07:47:40.794201+00:00/task_id=first_group.inner_function1 permission to 509
[2023-09-30T07:47:53.975+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=dags_python_with_task_group/run_id=scheduled__2023-09-29T07:47:40.794201+00:00/task_id=first_group.inner_function1 permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dags_python_with_task_group/run_id=scheduled__2023-09-29T07:47:40.794201+00:00/task_id=first_group.inner_function1'
[2023-09-30T07:47:54.427+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dags_python_with_task_group.first_group.inner_function1 scheduled__2023-09-29T07:47:40.794201+00:00 [queued]>
[2023-09-30T07:47:54.455+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dags_python_with_task_group.first_group.inner_function1 scheduled__2023-09-29T07:47:40.794201+00:00 [queued]>
[2023-09-30T07:47:54.456+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-30T07:47:54.561+0000] {taskinstance.py:1327} INFO - Executing <Task(_PythonDecoratedOperator): first_group.inner_function1> on 2023-09-29 07:47:40.794201+00:00
[2023-09-30T07:47:54.573+0000] {standard_task_runner.py:57} INFO - Started process 144 to run task
[2023-09-30T07:47:54.588+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dags_python_with_task_group', 'first_group.inner_function1', 'scheduled__2023-09-29T07:47:40.794201+00:00', '--job-id', '249', '--raw', '--subdir', 'DAGS_FOLDER/dags_python_with_task_group.py', '--cfg-path', '/tmp/tmpwezy9y2o']
[2023-09-30T07:47:54.594+0000] {standard_task_runner.py:85} INFO - Job 249: Subtask first_group.inner_function1
[2023-09-30T07:47:54.728+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=dags_python_with_task_group/run_id=scheduled__2023-09-29T07:47:40.794201+00:00/task_id=first_group.inner_function1 permission to 509
[2023-09-30T07:47:54.733+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=dags_python_with_task_group/run_id=scheduled__2023-09-29T07:47:40.794201+00:00/task_id=first_group.inner_function1 permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dags_python_with_task_group/run_id=scheduled__2023-09-29T07:47:40.794201+00:00/task_id=first_group.inner_function1'
[2023-09-30T07:47:54.740+0000] {task_command.py:410} INFO - Running <TaskInstance: dags_python_with_task_group.first_group.inner_function1 scheduled__2023-09-29T07:47:40.794201+00:00 [running]> on host 07fcf706bef2
[2023-09-30T07:47:55.019+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dags_python_with_task_group' AIRFLOW_CTX_TASK_ID='first_group.inner_function1' AIRFLOW_CTX_EXECUTION_DATE='2023-09-29T07:47:40.794201+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-29T07:47:40.794201+00:00'
[2023-09-30T07:47:55.024+0000] {logging_mixin.py:150} INFO - 첫 번째 TaskGroup 내 첫 번째 task입니다.
[2023-09-30T07:47:55.025+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-30T07:47:55.041+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=dags_python_with_task_group, task_id=first_group.inner_function1, execution_date=20230929T074740, start_date=20230930T074754, end_date=20230930T074755
[2023-09-30T07:47:55.138+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-30T07:47:55.206+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
